src/extension-runners/chromium.js

```diff
@@ -6,7 +6,7 @@
 import path from 'path';
 
 import fs from 'fs-extra';
-import asyncMkdirp from 'mkdirp';
```

Comment:
@willdurand this (and the similar tweak in artifacts.js) is the only change needed to fix the test failure (the changelog also confirms that mkdirp is not only exposing a named export).